src/components/vulnerability-summary/index.js

```diff
@@ -20,9 +20,11 @@ const VulnerabilityItem = (severity, vulnerabilitySummary) => {
       <div className="severity-summary__count">
         <span className="severity-summary__label">{severityCount}</span>
       </div>
-      <div className="severity-summary__text" title={abbrTitle}>
-        <span className="severity-summary__label">{abbrLabel}</span>
-      </div>
+      <Tooltip placement="auto" title={abbrTitle}>
```

Comment:
Use for example `placement="top"` to prevent different placements for different labels. 
Another suggestion: remove `cursor: help` styling. 

![image](https://user-images.githubusercontent.com/79976935/130759450-0ea29fe5-0dda-450b-a8f5-b07d213eccf9.png)

Reply:
fixed